ResearchKit.xcodeproj/project.pbxproj

```diff
@@ -2251,8 +2280,9 @@
 			isa = PBXHeadersBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				244EFAD21BCEFD83001850D9 /* ORKAnswerFormat_Private.h in Headers */,
```

Comment:
Should link `ORKAnswerFormat_Private.h` to RK as private membership to resolve current compilation failure.
<img width="132" alt="screen shot 2015-11-04 at 10 36 10 am" src="https://cloud.githubusercontent.com/assets/11466704/10947970/f78940d6-82df-11e5-8c62-40d7493b36ae.png">